### DISCUSSION TITLE
Encourage clients to use secp256k1 wallet for deal making

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/notary-governance#324] [Ideas]
The lotus team accidentally discovered that some clients are using bls (f3) wallet account to make deals. And using `bafy2bzaceblq7zpmavxtnahcmstnip5vrokw2w6hadmqewd3x47n23hgxbyce` as an example, it makes ~35% gas used for this PSD message is actually used for verifying client signature. However, if the client used a sect256k1 address, it can be 10x cheaper (see gas price list [here](https://github.com/filecoin-project/lotus/blob/14dcc477f1ba74a494f82ed811446ffd4b1090c5/chain/vm/gas.go#L106-L109)) and will save SP a lot of gas cost in PSD messages -> potentially leads to lower ask price for deal onboarding in general in the longer term. (also saves chan bandwidth a bit)

Therefore, for future datacap allocations, we'd like to propose to notary to encourage verified client to use secp256k1 address or consider even only allocate DataCap to secp256k1 address.

**Comment by jennijuju** [comment]
to be clear.. this is an immediate action that can be taken for https://github.com/filecoin-project/FIPs/issues/211 but def not the long term solution

**Comment by darrellhendershot** [comment]
where and how can I get this secp256k1 wallet?

**Reply by stuberman** [reply]
`lotus wallet new --help`

**Reply by Shekelme** [reply]
Or simply `lotus wallet new`. ))

**Comment by woshidama323** [comment]
As a miner, Is it necessary to change the wallet to a sect256k1 one?

**Comment by Shekelme** [comment]
Should we use secp256k1 wallets for other control addresses?

**Comment by Fatman13** [comment]
Are sect256k1 address just f1 or f4?
